File: java assignments/Day02assignment/src/com/sonata/day02assignment/Student.java
package com.sonata.day02assignment;
class stu
{
	int stdid;
	String stdname;
	String stdclass;
	stu(int a ,String b,String c){
		this.stdid = a;
		this.stdname = b;
		this.stdclass = c;
	}
	public void display() {
		System.out.println(stdid);
		System.out.println(stdname);
		System.out.println(stdclass);
	}
}

public class Student {

	public static void main(String[] args) {
	    stu s1 = new stu(10,"ganesh","abc");
	    stu s2 = new stu(20,"abhi","def");
	    s1.display();
	    s2.display();

	}

}

File: typescript examples/assignments/Student.js
var Student = /** @class */ (function () {
    function Student(i, n, c) {
        this.stdId = i;
        this.stdName = n;
        this.stdClass = c;
    }
    Student.prototype.display = function () {
        console.log(this.stdId);
        console.log(this.stdName);
        console.log(this.stdClass);
    };
    return Student;
}());
var s1 = new Student(1, "ganesh", "cse");
var s2 = new Student(2, "abhi", "eee");
s1.display();
s2.display();
